catkin_ws/src/stability/msg/wrench.msg

```diff
@@ -0,0 +1,2 @@
+geometry_msgs/Vector3 force
```

Comment:
We won't need this file or folder anymore

Reply:
ok, made the change